wikimedia/wikipedia-ios
# Address vanishing onboarding modal design feedback

**Phabricator:** https://phabricator.wikimedia.org/T320758#8342836

### Notes
* Updates vanish account view string
* Updates padding/spacing for vertical size class to display some body text when device is in landscape orientation

I could have missed something, but I didn't immediately see anything akin in UIKit's `flashScrollIndicators` for SwiftUI's `ScrollView`.

### Test Steps
1. On any iPhone simulator/device, log in and tap vanish account to see modal
2. Rotate device, confirm some body text of the modal is visible in landscape orientation
